### ISSUE TITLE
[SUGGESTION] Keybind manager in taskbar 

### ISSUE THREAD

**Issue by CynicalMuffin** [momentumdevelopment/cosmos#91]
Since a taskbar has been added and that i might as well suggest a dumb feature that is cool
Make a minecraft controls manager kinda thing for the cosmos modules 
The current keybinds method is fine but why not

**Comment by Sxmurai** [comment]
Apparently Linus wasn't done with the taskbar and he accidentally committed, rip

**Comment by Sxmurai** [comment]
Or atleast with the new gui changes

**Comment by linustouchtips** [comment]
> Apparently Linus wasn't done with the taskbar and he accidentally committed, rip

This is true, I am dyslexic and accidentally commited them with my NoSlow changes.

**Comment by CynicalMuffin** [comment]
at least its better than having to reloag onto the server to get the gui back lol
